doc/news/changes/minor/20200429Jean-PaulPelteret

```diff
@@ -0,0 +1,5 @@
+New: There is a new function FEValuesBase::quadrature_point_indices()
+that make writing range-based for loops over all quadrature points of a 
```

Comment:
makes

Reply:
Thanks for the review and for pointing that out. Fixed now!